README.md

```diff
@@ -3,7 +3,10 @@
 ## Development
 
 1. Install [pipenv](https://pipenv.pypa.io/en/latest/)
+1. Create a lock file that allows the black beta release: `pipenv lock --pre`
```

Comment:
Is this really needed every time some one creates a new clone of the repo? Or is it a one-time command that updated the lock file and now no one needs to run it again because the lockfile is changed appropriately?